.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: objective-c
-osx_image: xcode7.1
+osx_image: xcode8
 env:
   global:
-    - WORKSPACE="FluentConstraints.xcworkspace"
+    - PROJECT="FluentConstraints.xcodeproj"
     - SCHEME="FluentConstraints"
-    - SDK="iphonesimulator9.1"
+    - SDK="iphonesimulator10.0"
   matrix:
     - DESTINATION="OS=9.1,name=iPhone 5"
     - DESTINATION="OS=9.1,name=iPhone 5s"
     - DESTINATION="OS=8.4,name=iPhone 5"
     - DESTINATION="OS=8.4,name=iPhone 5s"
 script:
-  - xctool -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" test
+  - xctool -project "$PROJECT" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" test
```

Comment:
Appears that xctool has problems with Xcode 8. A possible workaround is at https://github.com/arouanet/Whimsy/commit/7b5c98b7320ef5ba1c8ec29ca0878249ffed0332.